packages/espressocash_app/integration_test/app_test.dart

```diff
@@ -16,39 +14,20 @@ void main() {
 
       // Tap on create wallet button.
       await tester.tap(find.byKey(keyCreateWalletButton));
-      await tester.pumpAndSettle();
-
-      // Submit button should have 'I Understand' text, tap on it.
-      await tester.tap(find.text('I Understand'));
-      await tester.pumpAndSettle();
-
-      // Ensure that recovery phrase is set.
-      final recoveryPhrase = find.byWidgetPredicate(
-        (widget) =>
-            widget is RecoveryPhraseTextView && widget.phrase.isNotEmpty,
-      );
-      expect(recoveryPhrase, findsOneWidget);
-
-      await tester.submit();
-
-      // Enter 'Sample' as a name.
-      await tester.enterText(find.byKey(keyCreateProfileName), 'Sample');
-      await tester.testTextInput.receiveAction(TextInputAction.done);
-      await tester.pumpAndSettle();
 
-      await tester.submit();
+      await tester.pumpManual(5);
 
-      // Ensure that account was created and logged in – puzzle reminder
-      // dialog will appear with 'Protect Your Wallet' title.
-      final confirmationDialogTitle = find.text('Protect Your Wallet');
-      expect(confirmationDialogTitle, findsOneWidget);
+      // Should display Investment Screen
+      final investmentScreen = find.byType(InvestmentsScreen);
+      expect(investmentScreen, findsOneWidget);
     });
   });
 }
 
 extension on WidgetTester {
-  Future<void> submit() async {
-    await tap(find.byKey(keyBottomButton));
-    await pumpAndSettle();
+  Future<void> pumpManual(int times) async {
+    for (int i = 0; i < times; i++) {
+      await pump(const Duration(seconds: 1));
```

Comment:
Why not just wait for the predefined duration instead of a loop?

Reply:
It's because it takes more than one pump to finish the transition, and we cannot use `pumpAndSettle` because of the rive animation making it pump indefinitely.